catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -443,4 +443,12 @@
         <bundle>mvn:org.apache.mina/mina-core/${mina.version}</bundle>
     </feature>
 
+    <feature name="catalog-async-inmemory" install="auto" version="${project.version}">
```

Comment:
This PR should include that Alliance change https://github.com/codice/alliance/pull/225